hockey/indivPlayerFunctions.php

```diff
@@ -0,0 +1,63 @@
+<?php
+function playerInfo($LastName){
+    $info = dbQuery("
+    SELECT *
+    FROM hockey_1617
+    WHERE LastName= :var1",
+    array(
+        "var1"=>$LastName)
+    );
+    return $info->fetch();
+};
+
+function displayPlayerInfo ($name){
+    if($_REQUEST['name']==$name){
```

Comment:
I'm not totally clear on what purpose this function serves. It seems like you're just calling `playerInfo` and returning the results. So why not just call `playerInfo` directly and get rid of this function?